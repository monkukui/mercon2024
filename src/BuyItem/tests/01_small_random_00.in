8
884999006 931368379 4
284938230 18042131 7
493751573 53310789 49
976862030 55058856 68
180141501 526470566 32
732251444 235251339 87
273051773 657195207 97
129237183 136293693 95
