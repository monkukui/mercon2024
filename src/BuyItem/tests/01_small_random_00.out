8
18042131 284938230
53310789 493751573
55058856 976862030
136293693 129237183
235251339 732251444
526470566 180141501
657195207 273051773
931368379 884999006
